dotcom-rendering/src/components/AdSlot.web.tsx

```diff
@@ -141,7 +141,7 @@ const adSlotCollapseStyles = css`
 	&.ad-slot.ad-slot--collapse {
 		display: none;
 	}
-	&.ad-slot--merchandising-billboard {
+	&.ad-slot--merchandising-hide-below-desktop {
```

Comment:
Would a generic class name be better here, in case other slots in the future want to be hidden below desktop?
Something like `&.ad-slot--hide-below-desktop`

Reply:
Or maybe `.ad-slot--collapse-below-desktop` would be more consistent with the above class

Reply:
Makes sense ill make the appropriate change in commercial PR too